src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/SealedClaimPdfTest.java

```diff
@@ -0,0 +1,77 @@
+package uk.gov.hmcts.cmc.claimstore.tests.functional;
+
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import org.junit.Before;
+import org.junit.Test;
+import org.pdfbox.pdmodel.PDDocument;
+import org.pdfbox.util.PDFTextStripper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import uk.gov.hmcts.cmc.claimstore.documents.ClaimIssueReceiptService;
+import uk.gov.hmcts.cmc.claimstore.idam.models.User;
+import uk.gov.hmcts.cmc.claimstore.tests.BaseTest;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.ClaimData;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Arrays;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SealedClaimPdfTest extends BaseTest {
+
+    private User claimant;
+    @Autowired
+    private ClaimIssueReceiptService claimIssueReceiptService;
+    static File fileName = new File("/Users/kiranv/cmc/claim-store/src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/999MC042-claim-form.pdf");
```

Comment:
to get the file need to download it from the document controller after the case has been created

Reply:
Hmm, expected something a bit different when I had a chat with @varma02 the other day. I though the approach was to have the logic to assert PDF content first (as it is easier) with assertions agreed with QA and then change the source of the PDF to take it from REST calls.

Reply:
shouldn't be much harder? 
we also have a story in this sprint to download sealed claim from the frontend so its needed functionality too

open to either way but i thought the agreed approach was to download the PDF after saving the claim?

Reply:
I had a chat with Vivek and we agreed to do asserts on Claim Ref, Issued on, Claimant name, Defendant name, Claim amount, Total amount, Respond before date and add other data later if needed.